package.json

```diff
@@ -91,7 +91,9 @@
     "randomatic": "^3.1.1",
     "logkitty": "^0.7.1",
     "minimist": "^1.2.5",
-    "npm": "^6.14.5",
+    "yargs-parser": "^13.1.2",
+    "npm": "^5.10.0",
```

Comment:
We recently upgraded the npm version due to a security alert from github. Can you check if this reintroduces that?

Reply:
Yes, I talked with @cmcewen (who made the security resolution), and he told me that it was ok.
I'm trying to see if there is a way to check this package from the github database, but I'm not finding any way

Reply:
Cool thanks for checking